client/my-sites/themes/themes-header.scss

```diff
@@ -0,0 +1,26 @@
+.themes__header {
+    display: flex;
+    flex: 1 1 auto;
```

Comment:
I believe this rule does nothing because `.themes_header` is not a flex child.

Reply:
This makes sense, I removed the property.